File: Examples/Ruby/Paragraphs/post_document_paragraph_run_font.rb
require 'aspose_words_cloud'

class WorkingWithParagraphs

  include AsposeWordsCloud
  include AsposeStorageCloud

  def initialize
    #Get App key and App SID from https://cloud.aspose.com
    AsposeApp.app_key_and_sid("APP_KEY", "APP_SID")
    @words_api = WordsApi.new  
  end

  def upload_file(file_name)
    @storage_api = StorageApi.new
    response = @storage_api.put_create(file_name, File.open("../data/" << file_name,"r") { |io| io.read } )
  end

  def post_document_paragraph_run_font
    file_name = "SampleWordDocument.docx"
    upload_file(file_name)

    font = Font.new
    font.name = "Arial"
    font.bold = true

    paragraph_index = 0
    run_index = 0
    response = @words_api.post_document_paragraph_run_font(file_name, paragraph_index, run_index, font)
  end

end

workingWithParagraphs = WorkingWithParagraphs.new()
puts workingWithParagraphs.post_document_paragraph_run_font

File: Examples/PHP/Paragraphs/PostDocumentParagraphRunFont.php
<?php
require_once realpath(__DIR__ . '/..') . '/vendor/autoload.php';
require_once realpath(__DIR__ . '/..') . '/Utils.php';

use Aspose\Words\WordsApi;
use Aspose\Words\AsposeApp;

class Paragraphs {

	public $wordsApi;

	public function __construct() {
		AsposeApp::$appSID = Utils::appSID;
		AsposeApp::$apiKey = Utils::apiKey;
		$this->wordsApi = new WordsApi();
	}


	public function postDocumentParagraphRunFont() {
    	// Upload file to Aspose Cloud Storage
		$fileName = "SampleWordDocument.docx";
		Utils::uploadFile($fileName);

        $body = '{
                    "Font:" {
                        "Name": "Arial",
                        "Bold": true
                    }
                }';
        $result = $this->words->PostDocumentParagraphRunFont($fileName, $index=0, $runIndex=0, $storage = null, $folder = null, $filename = null, $body);
        print_r($result);
    }
}

$paragraphs = new Paragraphs();
$paragraphs->postDocumentParagraphRunFont();

?>